lib/ReactViews/ExplorerWindow/Tabs/MyDataTab/my-data-tab.scss

```diff
@@ -136,9 +136,6 @@
 
 .btn--back-to-my-data {
   composes: btn from "../../../../Sass/common/_buttons.scss";
-  // TODO: fix?
-  // composes: btn-small from "../../../../Sass/common/_buttons.scss";
-  // composes: btn--left from "../../../../Sass/common/_buttons.scss";
```

Comment:
`btn--left` was an icon definition so it is safe to remove
`btn-small` wouldn't make any point here as we have all its styles defined/overriden here or in styled-component